horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/RunService.java

```diff
@@ -222,6 +222,13 @@ Response add(@QueryParam("test") String testNameOrId,
             @Parameter(name = "description", description = "Run description", example = "AWS runs"),
 
     })
+    @APIResponses(value = {
+            @APIResponse(responseCode = "202", description = "The request has been accepted for processing. Returns a list of created run IDs if available, "
+                    + "or an empty list if processing is still ongoing. Label values and change detection processing " +
+                    "is performed asynchronously.", content = @Content(mediaType = MediaType.TEXT_PLAIN, schema = @Schema(type = SchemaType.STRING, example = "101,102,103"), example = "101,102,103")),
```

Comment:
> I think that, as a follow up to this work, we should be returning APPLICATION_JSON instead of TEXT_PLAIN.

I totally agree with you!

Initially I started changing that as you're suggesting (i.e., always returning `APPLICATION_JSON`) but then the breaking change would have been much bigger. Therefore I reverted that back to `TEXT_PLAIN` to do one step at a time.